src/lib/mina_base/test/zero_vesting_period.ml

```diff
@@ -138,6 +138,21 @@ let mk_zkapp_with_vesting_period n =
                 "proved_state": [ "Ignore" ],
                 "is_new": [ "Ignore" ]
               },
+              "permissions": {
```

Comment:
This also seems to be in the wrong place, it should be in `account` above.

Reply:
Thanks for catching that, I moved the permissions inside account field elsewhere but forgot to move it here. It seems like that was causing the flaky proof cache issue.